source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -32,22 +32,16 @@ AI provider:
 - The schema of the collection you are querying, 
   including database name, collection name, field names, and types. 
 
-.. BEGIN-COMPASS-ONLY
```

Comment:
Removed these tags from this page since Atlas docs will have it's own version of this page that isn't migrated from Compass.